pkg/k8shandler/kibana/trustedcabundle.go

```diff
@@ -27,11 +29,26 @@ func (clusterRequest *KibanaRequest) createOrUpdateTrustedCABundleConfigMap(conf
 
 	utils.AddOwnerRefToObject(configMap, getOwnerRef(clusterRequest.cluster))
 
-	err := clusterRequest.CreateOrUpdateTrustedCaBundleConfigMap(configMap)
-	return err
+	err := clusterRequest.Create(configMap)
+	if err != nil {
+		if !errors.IsAlreadyExists(err) {
+			return nil, fmt.Errorf("failed to create trusted CA bundle config map %q for %q: %s", name, clusterRequest.cluster.Name, err)
+		}
+
+		// Get the existing config map which may include an injected CA bundle
+		if err = clusterRequest.Get(configMap.Name, configMap); err != nil {
```

Comment:
nit, but since we aren't doing anything with the CM other than ensuring it isn't nil (which should happen once we set it up for creating). not sure we need to do the extra api call ... we are already doing a `get` to build the hash in the `getAnnotations` function

Reply:
It's not valid to drop this `Get` call actually because the refreshed config map is used in building the pod spec. In the absence of a valid CA bundle the pod factory function does not add the volume/volumemount to the spec.

Reply:
ah, I didn't realize that `hasTrustedCABundle` ensures it's been populated
https://github.com/openshift/elasticsearch-operator/blob/master/pkg/k8shandler/kibana/trustedcabundle.go#L34-L44